examples/hello_world_util.h

```diff
@@ -284,7 +281,7 @@ static int barrier(int oob_sock)
     return !(res == sizeof(dummy));
 }
 
-static int generate_test_string(char *str, int size)
+static inline int generate_test_string(char *str, int size)
```

Comment:
why needed?

Reply:
better to keep it `static inline` to fix possible `defined, but not used` error

Reply:
why won't it be used?